docs/sources/reference/api/docker_remote_api_v1.19.md

```diff
@@ -1525,6 +1525,11 @@ Search for an image on [Docker Hub](https://hub.docker.com).
 > The response keys have changed from API v1.6 to reflect the JSON
 > sent by the registry server to the docker daemon's request.
 
+> **Note**:
```

Comment:
There are some English errors. Also , we don't want to stack notes like this.  We should make the  second note part of the description.   No need to mention we are keeping around `is_trusted` for backwards compatibility --- that is implied by the deprecation statement.

--> https://gist.github.com/moxiegirl/be45866159b906e3c773

Search for an image on [Docker Hub](https://hub.docker.com). This API
returns both `is_trusted` and `is_automated` images. Currently, they 
are considered identical. In the future, the `is_trusted` property will
be deprecated and replaced by the `is_automated` property.

> **Note**:
> The response keys have changed from API v1.6 to reflect the JSON
> sent by the registry server to the Docker daemon's request.